test/countAnimals.test.js

```diff
@@ -1,32 +1,43 @@
-const assert = require('assert');
-const zoo = require('../src/zoo');
+const countAnimals = require('../src/countAnimals');
 
-describe('Implemente a função countAnimals', () => {
-
-  it('Sem parâmetros, retorna animais e suas quantidades', () => {
-    const expected = {
-      'lions': 4,
-      'tigers': 2,
-      'bears': 3,
-      'penguins': 4,
-      'otters': 4,
-      'frogs': 2,
-      'snakes': 2,
-      'elephants': 4,
-      'giraffes': 6
-    };
-    const actual = zoo.countAnimals();
-
-    assert.deepStrictEqual(actual, expected);
-  });
-
-  it('Com o nome de uma espécie de animal, retorna somente a quantidade', () => {
-    let actual = zoo.countAnimals('lions');
-    let expected = 4;
-    assert.deepStrictEqual(actual, expected);
-
-    actual = zoo.countAnimals('snakes');
-    expected = 2;
-    assert.deepStrictEqual(actual, expected);
+describe('5 - Crie uma função que realiza a contagem dos animais de cada espécie', () => {
+  
+  describe('Implemente a função countAnimals', () => { 
+    it('Sem parâmetros, retorna as espécies e sua quantidade', () => {
+      const expected = {
+        'lions': 4,
+        'tigers': 2,
+        'bears': 3,
+        'penguins': 4,
+        'otters': 4,
+        'frogs': 2,
+        'snakes': 2,
+        'elephants': 4,
+        'giraffes': 6
+      };
+      const actual = countAnimals();
+  
+      expect(actual).toStrictEqual(expected);
+    });
+  
+    it('Recebendo como parâmetro um objeto com a chave \'specie\', retorna um número, a quantidade de animais daquela espécie', () => {
+      let actual = countAnimals({ specie: 'penguins' });
+      let expected = 4;
+      expect(actual).toBe(expected);
+  
+      actual = countAnimals({ specie: 'giraffes' });
+      expected = 6;
+      expect(actual).toBe(expected);
+    });
+  
+    it('Recebendo como parâmetro um objeto com a chave \'specie\' e \'genre\', retorna um número, a quantidade de animais daquela espécie, no gênero selecionado', () => {
```

Comment:
Caso queira mudar o enunciado ali em cima, vou deixar a sugestão aqui pra vc não precisa dar outro push pra mudar o nome do teste 

```suggestion
    it(Recebendo como parâmetro um objeto com a chave \`specie`\ e \`genre`\, retorna um número que é a quantidade de animais daquela espécie, no gênero selecionado, () => {
```